backend/flaskr/app.py

```diff
@@ -70,16 +70,18 @@ def rate_suggestion():
     """Apply the user's rating to their profile and the restaurant's"""
     args     = request.json.split('\n')
     name     = args[0]
-    is_liked = args[1]
+    rating   = args[1]
     rest_id  = args[2]
 
     user = _user(name)
 
     # Process rating
-    if is_liked:    # Add the accepted restaunt to the user's review list
+    if rating == "yummy":    # Add the accepted restaunt to the user's review list
         user.add_review(rest_id)
-    else:           # Otherwise send the disliked restaurant to the user's model for training
+    elif rating == "yuck":  # Send the disliked restaurant to the user's model for training
         user.disliked(rest_id)
+    elif rating == "maybe later": # User wants to go there but at a later time
+        pass # Do nothing but still cache the restaurant
```

Comment:
These conditionals need to match the arguments coming from FindQuiz.js. This PR uses "yummy" -> "1", "maybe later" -> "0", and yuck -> "-1". When I fixed this, the reviews came up properly on the review page. 

To fix, we can either change the conditionals here or change the arguments in FindQuiz.js. I would prefer changing the arguments since words are more descriptive than integers, but either way would work.